### PR TITLE
Fix for watchonly and nodes

### PR DESCRIPTION
**Pull request by TheTrunk**
This patch fixis watch only addresses. The user is notified by a messege whenever Hush Wallet is starting that a particular address is invalid or watch-only. It pop up seperately for each such an address. In Own addresses tab such instead of sucha an address a <INVALID OR WATCH-ONLY ADDRESS> !!! is displayed.  
However watch only addresses still counts towards total amount and user can select them in send tab.

I'm also adding nodes and fixing donation addresses.

### CODE REVIEW

**Comment by TheTrunk** [comment]
works2
works3
works

Both watch only address pop ups seperately, counts toward total amount and "can be" used/viewed in send tab.

**Comment by TheTrunk** [comment]
[HUSHSwingWalletUI-combined.zip](https://github.com/MyHush/hush-swing-wallet-ui/files/1670511/HUSHSwingWalletUI-combined.zip)
SHA256: A7A1ECAB46474114CA34F0382B2760CE1225387B9E7836129DC36B879D6F82BC

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2018 TheTrunk
```
Comment (joshgentry) [inline LICENSE]: If we want to change this, shouldn't this Copyright move under the Hush team?

File: src/java/com/vaklinov/zcashui/AddressesPanel.java [hunk 2]
```diff
@@ -376,6 +383,35 @@ private JTable createAddressBalanceTable(String rowData[][])
...
+			String addressToDisplay = address;
+			// Make sure the current address is not watch-only or invalid
+			if (!this.validationMap.containsKey(address))
+			{
+				boolean validationResult = this.clientCaller.isWatchOnlyOrInvalidAddress(address);
+				this.validationMap.put(address, new Boolean(validationResult));	
+				
+				if (validationResult)
+				{
+		            JOptionPane.showMessageDialog(
```
Comment (joshgentry) [inline src/java/com/vaklinov/zcashui/AddressesPanel.java]: Just a note that in this `if` block, you're mixing tabs and spaces

I might go through and normalize this throughout the project so I don't think it _needs_ to be changed, but an FYI

File: win/first-run.bat [hunk 3]
```diff
@@ -18,6 +21,10 @@ IF NOT EXIST %AppData%\Hush\hush.conf (
 
 IF NOT EXIST %LocalAppData%\HushSwingWalletUI\addressBook.csv (
    (
-    echo t1dSqorCh1PdyvViTRHzKdAFjnR81VyGv1u,HUSH Dev Fund
+    echo t1h6kmaQwcuyejDLazT3TNZfV8EEtCzHRhc,HUSH Donation address
+	echo t1dxPQTkv3j6ifqRKQRkMpBrDyhKWm3WXma, TheTrunk
```
Comment (joshgentry) [inline win/first-run.bat]: I don't think including individual dev addresses in the wallet addressbook bootstrap is beneficial to the project, the readme would be a better spot for these

Dev team address is fine

File: LICENSE (hunk 1 above)
Comment (TheTrunk) [reply LICENSE]: Agree

File: win/first-run.bat (hunk 3 above)
Comment (TheTrunk) [reply win/first-run.bat]: Agree

File: src/java/com/vaklinov/zcashui/AddressesPanel.java (hunk 2 above)
Comment (TheTrunk) [reply src/java/com/vaklinov/zcashui/AddressesPanel.java]: It's mixed in every file. :)

**Review by joshgentry — APPROVED** [review]
LGTM 👍

**Comment by nimbosa** [comment]
👍

**Comment by TheTrunk** [comment]
Do not forget to change the donation dev address! The donation address is the old hush team donation address that is in possesion of madbuda who is no loger part of the hush team. You should also update the nodes.

**Pull request merged**